clients/drcachesim/tools/view.h

```diff
@@ -115,11 +115,11 @@ class view_t : public analysis_tool_t {
     virtual bool
     init_decode_cache();
 
-    // Initializes various other state based on the filetype. If the
-    // TRACE_MARKER_TYPE_FILETYPE was not seen, the filetype is acquired from the
-    // memtrace_stream_t object instead.
+    // Initializes various other state based on the filetype and trace version. If the
```

Comment:
I don't see any state initialized based on the version.  See the other suggestion to split this into two functions.